docs/supported-tools.rst

```diff
@@ -0,0 +1,246 @@
+
+Conda:
+   Miniconda2 ``4.6.14``
+   (could be updated to use the latest version by default).
+
+Mkdocs:
+  Latest version by default installed via ``conda``.
+
+Sphinx:
+  Latest version by default installed via ``conda``.
+
+sphinx-rtd-theme:
+  Latest version by default installed via ``conda``.
+
+mock:
+  Latest version by default installed via ``pip`` (could be removed in the future).
+
+pillow:
+  Latest version by default installed via ``pip`` (could be removed in the future).
+
+recommonmark:
+  Latest version by default installed via ``conda`` (could be removed in the future).
+
+Internal dependencies
+~~~~~~~~~~~~~~~~~~~~~
+
+Internal dependencies are needed to integrate your docs with Read the Docs.
+We guarantee that these dependencies will work with all current supported versions of our tools,
+you don't need to specify them in your requirements.
+
+- readthedocs-sphinx-ext
+
+Table of supported versions
+---------------------------
+
+Sphinx
+~~~~~~
+
+Sphinx releases its versions incrementally.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Version
+     - Released / Latest Update
+     - Supported Until
+
+   * - ``3.x``
+     - Apr 5, 2020 / ???
+     - 5.0 is released or later/early
+
+   * - ``2.x``
+     - Mar 28, 2019 / Mar 5, 2020
+     - 4.0 is released or later/early
+
+   * - ``1.8.x``
+     - Sep 12, 2018 / Mar 10, 2019
+     - Nov 31, 2022
+
+   * - ``1.7.x``
+     - Feb 12, 2018 / Sep 5, 2018
+     - Nov 31, 2022
+
+   * - ``1.6.x``
+     - May 16, 2017 / Feb 4, 2017
+     - Nov 31, 2021
+
+   * - ``1.5.x``
+     - Dec 5, 2016 / May 4, 2017
+     - Nov 31, 2021
+
+   * - ``<= 1.4.x``
+     - Mar 21, 2008 / Nov 23, 2016
+     - Unsupported
+
+Mkdocs
+~~~~~~
+
+MkDocs releases its versions incrementally.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Version
+     - Released / Latest Update
+     - Supported Until
+
+   * - ``1.1.x``
+     - Feb 22, 2020 / ???
+     - 3.0 released or later/early
+
+   * - ``1.0.x``
+     - Aug 3, 2018 / Sep 17, 2018
+     - 2.0 released or later/early
+
+   * - ``0.17.x``
+     - Oct 19, 2017 / Jul 6, 2018
+     - Nov 31, 2021
+
+   * - ``0.16.x``
+     - Nov 4, 2017 / Apr 4, 2017
+     - Nov 31, 2021
+
+   * - ``0.15.x``
+     - Jan 21 2016 / Feb 18, 2016
+     - Nov 31, 2021
+
+   * - ``<= 0.14.x``
+     - Jan 11, 2014 / Jun 9, 2015
+     - Unsupported
+
+Python
+~~~~~~
+
+Python defines an EOL (End Of Life) date for all its versions.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Version
+     - EOL Date
+     - Supported Until
+
+   * - ``3.9.x`` (not available yet)
+     - Oct 05, 2025
+     - Jan 31, 2026
+
+   * - ``3.8.x``
+     - Oct 14, 2024
+     - Jan 31, 2025
+
+   * - ``3.7.x``
+     - Jun 27, 2023
+     - Sep 31, 2023
+
+   * - ``3.6.x``
+     - Dec 23, 2021
+     - Mar 31, 2022
+
+   * - ``3.5.x``
+     - Sep 13, 2020
+     - Nov 31, 2021
+
+   * - ``2.7.x``
+     - Jan 01, 2020
+     - Nov 31, 2021
```

Comment:
This is overdue... but we didn't have this policy before... so

Reply:
It's also a huge change, and will likely still lead to a lot of yelling.